q.yaml

```diff
@@ -0,0 +1,3 @@
+Resources:
+  NotAResource:
+    Type: AWS::S3::Busket
```

Comment:
E3001: Invalid or unsupported Type AWS::S3::Busket for resource NotAResource in us-east-1  <!-- Automatically posted by Lintly -->